salt/utils/crypt.py

```diff
@@ -11,14 +11,21 @@
 
 log = logging.getLogger(__name__)
 
-
 try:
-    import Crypto.Random  # nosec
+    from Cryptodome import Random
```

Comment:
We should be checking for m2crypto and using that first. There shouldn't be a call to `Random.at_fork` when m2crypto is installed. 

The logic in this file should match here:

https://github.com/saltstack/salt/blob/master/salt/crypt.py#L45